recipes/hwloc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
while not strictly necessary syntax-wise in this file, we are trying to make sure new recipes follow the template  - otherwise it's very likely that new PRs that touch this recipe will amend this, adding unnecessary review burden

please make sure this lists 3.15:
https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_package/CMakeLists.txt#L1